storage/block/block.go

```diff
@@ -217,7 +214,6 @@ func (b *dbBlock) Stream(blocker context.Context) (xio.BlockReader, error) {
 		return xio.EmptyBlockReader, errReadFromClosedBlock
 	}
 
-	b.ctx.DependsOn(blocker)
```

Comment:
this method too, can you just drop the context arg?

Reply:
We need it for if it goes to the retriever (which takes a ctx to register some finalizations).